tables/yi.utb

```diff
@@ -0,0 +1,234 @@
+# liblouis: Uncontracted braille table for Yiddish.
+#
+#
+# This file is part of liblouis.
+#
+# liblouis is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1 of the
+# License, or (at your option) any later version.
+#
+# liblouis is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with liblouis. If not, see
+# <http://www.gnu.org/licenses/>.
+
+#-index-name: Yiddish, uncontracted
+#-display-name: Yiddish uncontracted braille
```

Comment:
I suggest dropping the "uncontracted". This is what we do with all "uncontracted" tables that don't have a "contracted" table in Liblouis. Are there contractions in Yiddish braille?